chapters/en/chapter7/speech-to-speech.mdx

```diff
@@ -0,0 +1,259 @@
+
+In the remainder of this Unit, we'll focus on creating a STST system that translates speech from any language X to speech
+in English. The methods covered can be extended to STST systems that translate from any language X to any 
+langauge Y, but we leave this as an extension to the reader and provide pointers where applicable. We further divide up the 
+task of STST into its two constituent components: ST and TTS. We'll finish by piecing them together to build a Gradio 
+demo to showcase our system.
+
+## Speech translation
+
+We'll use the Whisper model for our speech translation system, since it's capable of translating from over 96 languages
+to English. Specifically, we'll load the [Whisper Base](https://huggingface.co/openai/whisper-base) checkpoint, which 
+clocks in at 74M parameters. It's by no means the most performant Whisper model, with the [largest Whisper checkpoint](https://huggingface.co/openai/whisper-large-v2)
+being over 20x larger, but since we're concatenating two auto-regressive systems together (ST + TTS), we want to ensure 
+each model can generate relatively quickly so that we get reasonable inference speed:
+
+```python
+import torch
+from transformers import pipeline
+
+device = "cuda:0" if torch.cuda.is_available() else "cpu"
+pipe = pipeline(
+    "automatic-speech-recognition", model="openai/whisper-base", device=device
+)
+```
+
+Great! To test our STST system, we'll load an audio sample in a non-English language. Let's load the first example of the
+Italian (`it`) split of the [VoxPopuli](https://huggingface.co/datasets/facebook/voxpopuli) dataset:
+
+```python
+from datasets import load_dataset
+
+dataset = load_dataset("facebook/voxpopuli", "it", split="validation", streaming=True)
+sample = next(iter(dataset))
+```
+
+To listen to this sample, we can either play it using the dataset viewer on the Hub: [facebook/voxpopuli/viewer](https://huggingface.co/datasets/facebook/voxpopuli/viewer/it/validation?row=0)
+
+Or playback using the ipynb audio feature:
+
+```python
+from IPython.display import Audio
+
+Audio(sample["audio"]["array"], rate=sample["audio"]["sampling_rate"])
+```
+
+Now let's define a function that takes this audio input and returns the translated text. You'll remember that we have to 
+pass the generation key-word argument for the `"task"`, setting it to `"translate"` to ensure that Whisper performs 
+speech translation and not speech recognition:
+
+```python
+def translate(audio):
+    outputs = pipe(audio, max_new_tokens=256, generate_kwargs={"task": "translate"})
+    return outputs["text"]
+```
+
+<Tip>
+    Whisper can also be 'tricked' into translating from speech in any language X to any language Y. Simply set the task to
+    <code>"transcribe"</code> and the <code>"language"</code> to your target language in the generation key-word arguments, 
+    e.g. for Spanish, one would set:
+    <p><code>generate_kwargs={"task": "transcribe", "language": "es"}</code></p>
+</Tip>
```

Comment:
```suggestion
<Tip>

    Whisper can also be 'tricked' into translating from speech in any language X to any language Y. Simply set the task to
    `"transcribe"` and the `"language"` to your target language in the generation key-word arguments, 
    e.g. for Spanish, one would set:

	`generate_kwargs={"task": "transcribe", "language": "es"&rcub;`

</Tip>
```

fix the parsing issue. You can have extra newlines after `<Tip>` & before `</Tip>` to use markdown. Therefore, you can just use "`" instead of "<code>"